frontend/src/metabase/static-viz/index.js

```diff
@@ -1,9 +1,15 @@
 import React from "react";
 import ReactDOMServer from "react-dom/server";
 import StaticChart from "./containers/StaticChart";
+import { colors, color } from "metabase/lib/colors/palette";
 
 export function RenderChart(type, options) {
   return ReactDOMServer.renderToStaticMarkup(
     <StaticChart type={type} options={options} />,
   );
 }
+
+export function color_from_name(name) {
```

Comment:
Based on the existing function name case, we should follow them too. And it isn't like this function is called from BE that it should be named in the snake case.